src/System.IO.Pipes/tests/NamedPipeTests/NamedPipeTest.CurrentUserOnly.netcoreapp.cs

```diff
@@ -41,6 +41,19 @@ public static void CreateServer_ConnectClient()
             }
         }
 
+        [Fact]
+        public static void CreateServer_ConnectClient_UsingUnixAbsolutePath()
+        {
+            string name = Path.Combine("/tmp", GetUniquePipeName());
```

Comment:
I guess this assumes `/tmp` exists? I think it might not and you should use `$TMPDIR` ?

Reply:
Our code actually defaults to `/tmp` if `$TMPDIR` was not defined. That was the case in my vanilla Ubuntu 16.04 t so I assumed that we are already falling back to it in many places (macOS does define `$TMPDIR` and my vanilla one has the `/tmp` directory).

https://github.com/dotnet/coreclr/blob/f1fee6d7b4d06b17d872fdf587a4f87ca69827c2/src/mscorlib/shared/System/IO/Path.Unix.cs#L72